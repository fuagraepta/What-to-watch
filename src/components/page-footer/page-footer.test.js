import React from 'react';
import renderer from 'react-test-renderer';

import PageFooter from './page-footer.jsx';

describe(`The component is rendered correctly`, () => {
  it(`Page footer correctly renders`, () => {
    const pageFooter = renderer
      .create(<PageFooter/>)
      .toJSON();

    expect(pageFooter).toMatchSnapshot();
  });
});
