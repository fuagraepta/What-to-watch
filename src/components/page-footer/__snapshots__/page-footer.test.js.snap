// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The component is rendered correctly Page footer correctly renders 1`] = `
<footer
  className="page-footer"
>
  <div
    className="logo"
  >
    <a
      className="logo__link logo__link--light"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="copyright"
  >
    <p>
      Ã‚Â© 2019 What to watch Ltd.
    </p>
  </div>
</footer>
`;
